domino_cost/cost.py

```diff
@@ -48,6 +63,7 @@ def distribute_cost(df: DataFrame) -> DataFrame:
     """
     distributes __unallocated__ cost for cleaner representation.
     """
+    logger.info("distribute costs")
```

Comment:
Do we need the logging to be info in all of them?

Also, I'm seeing a couple of warnings:
```
09/04/2024 3:05 PM : /mnt/domino_cost/cost.py:73: SettingWithCopyWarning:
09/04/2024 3:05 PM : A value is trying to be set on a copy of a slice from a DataFrame.
09/04/2024 3:05 PM : Try using .loc[row_indexer,col_indexer] = value instead
09/04/2024 3:05 PM : See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
```